### PR TITLE
fixes and enhancements

### PR DESCRIPTION
**Pull request by harishnf**
Fixes:
page Style changes (page background colour, gradient, Font size, Filter text box background color, Input box size increase).
Promises code fix to execute asynchronous execution.  
reset tree to close all opened nodes.

Code Cleanup: 
 removed unused code. Example: drawGraphTooltip function.
 appwan name modified to service policies.

Enhancements:

added config class to show configs to a service
show service config in the tooltip

Service policy node —> added promises/api code to show Dial Services

Service —> fixed defect in Bind Identities logic
                    added promises/api code to show Associated Routers
                    added promises/api code to show  Service Configs
                    added promises/api code to show  Dial Identities

Identities —> added promises/api code to show Associated Routers
                       added promises/api code to show  Bind Services
                       added promises/api code to show  Dial Services

Router Policies —>  added promises/api code to show  Associated Routers

Search feature enhancements:
  In the new design, the visualizer can fetch a maximum of 1000 objects per network component. Whenever there are more than 1000 objects, we will show visualizer with the total object count for each node, the user can use the search feature to pull them based on filter criteria. Although the search API returned hundreds of records based on the filter text, the autocomplete list will only display the top five records..  ex: search word:'ha', result: top 5 of 362 records.

image
image
image

### CODE REVIEW

**Comment by harishnf** [comment]
This PR has the following changes.

Fixes:
page Style changes (page background colour, gradient, Font size, Filter text box background color, Input box size increase).
Promises code fix to execute asynchronous execution.  
reset tree to close all opened nodes.

Code Cleanup: 
I removed unused code. Example: drawGraphTooltip function.
Edge router, edge router policies, and appwan names are modified to the router, router policies and service policies.


Enhancements:

added config class to show configs to a service
show service config in the tooltip

Service policy node —> added promises/api code to show Dial Services

Service —> fixed defect in Bind Identities logic
                    added promises/api code to show Associated Routers
                    added promises/api code to show  Service Configs
                    added promises/api code to show  Dial Identities

Identities —> added promises/api code to show Associated Routers
                       added promises/api code to show  Bind Services
                       added promises/api code to show  Dial Services

Router Policies —>  added promises/api code to show  Associated Routers

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
     padding-left: 3px;
     padding-right: 3px;
     background: #f5e9f2;
-    background-image: linear-gradient(#ecd3e6 2px, transparent 0), linear-gradient(90deg, #ecd3e6 2px, transparent 0), linear-gradient(#ecd3e6 1px, transparent 0), linear-gradient(90deg, #ecd3e6 1px, transparent 0);
+    background-image: linear-gradient(#ecd3e6 0.5px, transparent 0), linear-gradient(90deg, #ecd3e6 1px, transparent 0), linear-gradient(#ecd3e6 0.2px, transparent 0), linear-gradient(90deg, #ecd3e6 0.5px, transparent 0);
```
Comment (rgallettonf) [inline projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss:30]: typically using decimals  for `px` values is an anti-patern. If we really need to be that granular, we should use `rem`

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss [hunk 2]
```diff
@@ -115,7 +115,7 @@ select {
 
 #ClearSearchFilter {
     position: absolute;
-    right: 2px;
+    right: -45px;
```
Comment (rgallettonf) [inline projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss:118]: Negative orientation for `left/right` styling is also an anti-pattern. Usually indicates a larger issue with the layout of the elements container.

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss [hunk 3]
```diff
@@ -216,8 +217,8 @@ select:focus {
 
 @supports ((-webkit-backdrop-filter: saturate(110%) blur(15px)) or (backdrop-filter: saturate(110%) blur(15px))) {
     .nf-topology-filter-input {
-        background: var(--glass) !important;
-        -webkit-backdrop-filter: saturate(110%) blur(15px);
+        background: #ffffff80 !important;
```
Comment (rgallettonf) [inline projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss:220]: This should probably remain as a css variable. You can add application level css variables to the `styles.scss` stylesheet.

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss (hunk 2 above)
Comment (harishnf) [reply projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss]: will address in my next PR

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss (hunk 1 above)
Comment (harishnf) [reply projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss]: changed px to pt

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss (hunk 3 above)
Comment (harishnf) [reply projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss:221]: will address this issue in next PR

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss (hunk 1 above)
Comment (rgallettonf) [reply projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss]: Resolved this with latest commit

File: projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss (hunk 3 above)
Comment (rgallettonf) [reply projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss]: resolved with latest commit

**Review by rgallettonf — APPROVED** [review]

**Pull request merged**